services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -76,16 +77,12 @@ const TwoColumnDetails = ({
         direction="row"
         className="detail-grid-item"
       >
-        {firstColumn && hasPermission(firstColumn) && (
-          <Grid xs={12} sm={5} item className="detail-grid-item">
-            <DetailItem item={firstColumn} />
-          </Grid>
-        )}
-        {secondColumn && hasPermission(secondColumn) && (
-          <Grid xs={12} sm={5} item className="detail-grid-item">
-            <DetailItem item={secondColumn} />
-          </Grid>
-        )}
+        <Grid xs={12} sm={5} item className="detail-grid-item">
+          {hasPermission(firstColumn) && <DetailItem item={firstColumn} />}
+        </Grid>
+        <Grid xs={12} sm={5} item className="detail-grid-item">
+          {hasPermission(secondColumn) && <DetailItem item={secondColumn} />}
+        </Grid>
```

Comment:
i like this change to always declare the grid element and only render inside if you have permission to view the element! Nice way to keep the defined position!